README.md

```diff
@@ -44,10 +44,12 @@ Minimum configuration:
 ```
 
 Configuration options:
-* `rbac`* - Full path to your running RBAC service. The rbac url should contain an `{addr}` string representing the wallet address.
+* `rbac`* - Full path to your running RBAC service. The rbac url should contain an `{addr}` string representing the wallet address. If empty string `""` then plugin will only verify a user signed JWT and will not get delegated access rights or include the x-wallet-access header. 
+The `""` option is included as a means to use the plugin to only verify the user signed JWT. 
 * `apikey` - API Key to use when making a call to the RBAC service
 * `authHeader` - The name of the request header containing the JWT. Defaults to "Authorization"
 * `accessHeader` - The name of the header to inject with the wallet access JSON. Defaults to "x-wallet-access"
+* `includeSenderHeader` - Option to return the `x-sender` header which is the addr claim of the user signed JWT. Defaults to false
```

Comment:
What's the use case for this option, and why is it `false` by default?

Reply:
I answered my own question: This is for our use case of proving ownership of the wallet address, and would contain that wallet address.

I'd suggest instead of on/off we allow a configurable name similar to the other header configuration options. We can debate whether when no value is specified we should default to `x-sender` or simply not add the header as this does.

Reply:
I set this configuration similar to the other header configuration options. On default I defaulted to not add any header